orocos_kdl/src/chainhdsolver_vereshchagin_doc.md

```diff
@@ -0,0 +1,206 @@
+
+# Documentation on the Vereshchagin Hybrid Dynamics solver
+#### Author of this document: Djordje Vukcevic (GitHub account: `djoleMNE`)
+
+## This solver is also known under the following names:
+* Acceleration Constrained Hybrid Dynamics (ACHD) solver
+* Popov-Vereshchagin solver
+---------------------
+
```

Comment:
White space can be limited. Maybe good idea to run a markdown linter.

Reply:
I will run markdown linter on it.